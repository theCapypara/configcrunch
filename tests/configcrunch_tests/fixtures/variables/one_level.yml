base:
  str_field: "{{ more.a }}"
  int_field: 12
  more:
    a: "{{ more.b }}"
    b: b
    c: "{{ int_field }}"
    d: "{{ int_field|str }}"
    in_text: "This is the alphabet: a {{ more.a }} c d"