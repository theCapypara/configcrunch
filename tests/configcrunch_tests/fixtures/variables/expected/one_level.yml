base:
  str_field: b
  int_field: 12
  more:
    a: b
    b: b
    c: 12
    d: '12'
    in_text: "This is the alphabet: a b c d"